tests/TestCase/View/Helper/FormHelperTest.php

```diff
@@ -7413,6 +7413,41 @@ public function testDateTimeWithFractional()
         $this->assertHtml($expected, $result);
     }
 
+    /**
+     * testControlWithFractional method
+     *
+     * Test that control() works with datetimefractional.
+     *
+     * @return void
+     */
+    public function testControlWithFractional()
+    {
+        $this->Form->create([
+            'schema' => [
+                'created' => ['type' => 'datetimefractional'],
+            ],
+        ]);
+        $result = $this->Form->control('created', [
+            'val' => new FrozenTime('2019-09-27 02:52:43'),
```

Comment:
Should this value have microseconds to ensure they work their way down into the value?